test/features/user_interacts_with_an_application.rb

```diff
@@ -0,0 +1,41 @@
+require './test/test_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.